config/settings.py

```diff
@@ -74,6 +74,7 @@
     'authentication',
     'storages',
     'contact_emails',
+    'weekly_digest',
```

Comment:
If you don't include the app here the command `python manage.py makemigrations` won't pick up new models.